### PR TITLE
chore(e2e): uniting testid attributes

### PR DESCRIPTION
**Pull request by ondracja**
I've united the usage of testid attributes for Suite web and Suite desktop tests under one version -> `data-testid`.

I've also added the setting into all other playwright configs (the connect ones) in case somebody starts to use those.

### CODE REVIEW

**Review by karliatto — COMMENTED** [review]
Is there any reason to use `data-testid` instead of `data-test`? 
It seams to me that `data-test` is more commonly used.
Also it looks like it does not follow proper kebapcase,  shouldn't it be `data-test-id` in any case?

**Comment by ondracja** [comment]
> Is there any reason to use `data-testid` instead of `data-test`? It seams to me that `data-test` is more commonly used. Also it looks like it does not follow proper kebapcase, shouldn't it be `data-test-id` in any case?

Hmm, good question @karliatto . 

I've been using `data-testid` from the beginning as that's what a framework's method is usually called and what is used in other repos. 

I'd like to keep denoting it with `id` since there should be no doubts about its purpose but I will refactor it in proper kebab case to keep it consistent as I'm using something like `data-test-activated` in some tests.

**Comment by karliatto** [comment]
> > Is there any reason to use `data-testid` instead of `data-test`? It seams to me that `data-test` is more commonly used. Also it looks like it does not follow proper kebapcase, shouldn't it be `data-test-id` in any case?
> 
> Hmm, good question @karliatto .
> 
> I've been using `data-testid` from the beginning as that's what a framework's method is usually called and what is used in other repos.
> 
> I'd like to keep denoting it with `id` since there should be no doubts about its purpose but I will refactor it in proper kebab case to keep it consistent as I'm using something like `data-test-activated` in some tests.

Ok, that looks weird to me since I always have seen `data-test` but I see they use it in documentation as well https://playwright.dev/docs/api/class-page#page-get-by-test-id

I do not really have hard opinions on that and I like consistency so whatever you like I am OK with that if the tests are passing.

**Comment by Nodonisko** [comment]
FYI on mobile only data-testid is valid nothing other works

**Comment by ondracja** [comment]
> FYI on mobile only data-testid is valid nothing other works

Damn, I thought it is just `testID` and thus is unaffected by the blanket chage. Will refactor this back.

**Comment by Nodonisko** [comment]
@ondracja Yea, sorry my bad it's `testID` https://reactnative.dev/docs/view#testid

File: submodules/trezor-common
Comment (peter-sanderson) [inline submodules/trezor-common:1]: Beware! This is here by accident!

**Comment by ondracja** [comment]
Closing as this PR is now obsolete. Will revisit this topic at a later point.

**Pull request closed without merging**